code/src/hooks/use-user-cart-items/useUserCartItems.ts

```diff
@@ -1,26 +1,56 @@
-import useGetCart from "@/hooks/use-get-cart/useGetCart";
-import useRemoveFromCart from "@/hooks/use-remove-from-cart/useRemoveFromCart";
-import { useUserDetailsSelector } from "@/store/slices/userSlice";
-import { CartItem } from "@/types/cart.types";
-
-const useUserCartItems = () => {
-  const user = useUserDetailsSelector();
-
-  const { data: cartItems, isError, isLoading: cartItemsLoading } = useGetCart();
-
-  const [removeItem] = useRemoveFromCart();
-
-  const handleRemoveItem = async (product: CartItem) => {
-    removeItem(product);
-  };
-
-  return {
-    user: user,
-    cartItems,
-    cartItemsLoading,
-    isError,
-    handleRemoveItem
-  };
-};
-
-export default useUserCartItems;
+import useGetCart from "@/hooks/use-get-cart/useGetCart";
+import useRemoveFromCart from "@/hooks/use-remove-from-cart/useRemoveFromCart";
+import useUpdateCartItemQuantity from "@/hooks/use-update-cart-item-quantity/useUpdateCartItemQuantity";
+import { useUserDetailsSelector } from "@/store/slices/userSlice";
+import { CartItem } from "@/types/cart.types";
+
+const useUserCartItems = () => {
+  const user = useUserDetailsSelector();
+
+  const {
+    data: cartItems,
+    isError,
+    isLoading: cartItemsLoading
+  } = useGetCart();
+
+  const [removeItem] = useRemoveFromCart();
+  const {
+    updateQuantity,
+    isLoading: updating,

```

Comment:
change name to for example isUpdating. It is a little bit confusing because it is bolean variable

Reply:
> change name to for example isUpdating. It is a little bit confusing because it is bоolean variable

Changed